packages/wrapped-keys-lit-actions/src/lib/ethereum/internal/signTransaction.js

```diff
@@ -0,0 +1,141 @@
+/* global ethers, Lit */
+
+function getValidatedUnsignedTx(unsignedTransaction) {
+  try {
+    if (!unsignedTransaction.toAddress) {
+      throw new Error('Missing required field: toAddress');
+    }
+
+    if (!unsignedTransaction.chain) {
+      throw new Error('Missing required field: chain');
+    }
+
+    if (!unsignedTransaction.value) {
+      throw new Error('Missing required field: value');
+    }
+
+    if (!unsignedTransaction.chainId) {
+      throw new Error('Missing required field: chainId');
+    }
+
+    return {
```

Comment:
Moved hexlify call to this so that we get errors about invalid parsing before running the rest of the LA code like fetching nonces etc.